templates/uimodules/image.html

```diff
@@ -75,13 +75,40 @@ <h1>{{escape(sharedfile.get_title())}}</h1>
                             width="{{sized_width}}"
                             loading="lazy"
                             decoding="async"
-                            alt="{{escape(sharedfile.get_title())}}"></a>
+                            alt="{{escape(sharedfile.get_alt_text(raw=True) or 'No alt from user')}}"></a>
                     {% end %}
                 {% end %}
             {% end %}
         {% end %}
     </div>
     <div id="image-content-footer-{{sharedfile.share_key}}" class="image-content-footer">
+        <div class="alt-text {% if can_edit %}alt-text-edit{%end%}{% if not sharedfile.get_alt_text() %} alt-text--blank{%else%} alt-text--hidden{%end%}">
+            <div class="alt-text-toggle link link--primary">alt text</div>
+            {% if can_edit and not site_is_readonly %}
+                {% if not sharedfile.get_alt_text() %}
+                    <div class="the-alt-text">add some alt text</div>
+                {% else %}
+                    <div class="the-alt-text">{{sharedfile.get_alt_text()}}</div>
+                {% end %}
+                <form method="post" class="alt-text-edit-form" action="/p/{{sharedfile.share_key}}/quick-edit-alt-text">
+                    {{ xsrf_form_html() }}
+                    <h3>Edit alt text</h3>
+                    <p class="alt-text-learn">
+                      <a href="/faq/#alt-text">learn about alt text</a>
```

Comment:
Here's a suggestion, let's use [the alt text hint from Bluesky](https://bsky.app/profile/bsky.app/post/3jzaiwxn7oy2n):

```suggestion
                      Alt text describes images for blind and low-vision users, and helps give context to everyone. <a href="/faq/#alt-text">Learn more</a>.
```